### ISSUE TITLE
feat: click on instance and highlight in metadata

### ISSUE THREAD

**Issue by willeppy** [cmudig/Texture#4]
Select a point from the table view and annotate the metadata charts on where this point sits.

Could also be useful as part of broader "instance view" workflow to 
(1) find similar points
(2) see where currently selected point lies in the metadata distributions

**Comment by domoritz** [comment]
Yeah, would be nice to be able to click on words or authors to then select those in the filters.
